### ISSUE TITLE
 Spotlight Top GitHub Repos for Egyptian Tech Community

### ISSUE THREAD

**Issue by iahmedgamal** [egytech-fyi/egytech-fyi#38]
## Objective
Develop a webpage to spotlight top GitHub repositories relevant to the Egyptian tech community.

## Tasks
1. Research and identify significant GitHub repositories benefiting the Egyptian market, feel free to add yours as well as comments. 
2. Design a straightforward page layout for displaying selected repositories.
3. Implement the page with the repository list.

## Criteria for Repository Selection
- Relevance to Egyptian tech market or community
- Active maintenance and community involvement
- Impact and usefulness

## Question
Do we need to add this new page to the frontend repository? Please consider if it aligns with the scope of our frontend repo.

## Refrence 
- https://awesome-morocco.dev/open-source
## Inspired by 
- @mahsayedsalem  
- also from @geeksblabla

**Comment by mahsayedsalem** [comment]
Can you lead this?

**Comment by iahmedgamal** [comment]
Yes I can lead this feature until we have it live in the site, this task is separated into two task, the first one is doing the research to 
collect the most interesting GitHub repos, the second task is just implement POC to show those in a list in a new page in 
`egytech-fyi` site. 

The next step is to have the repos in the BE or collect this information from the survey directly, but I prefer for now to do it manually by collecting the repos by ourselves, then we can think in more dynamic way. 

Also, if you have a discord channel invite me or post the link here, it's going to be easier for the collaborators to communicate better. 

One last thing, if you have a recommendation for any repo you want to see in this list, just add it here for now. 

🫡

**Comment by 0xMohamed** [comment]
@iahmedgamal, if you will be working on this feature, I would be happy to contribute and assist you if needed.

**Comment by iahmedgamal** [comment]
@0xMohamed Hello Mohamed, 

Yes, I'm planning to work on this page, let's connect to think about it together,
I will send you a message on Twitter

**Comment by iahmedgamal** [comment]
Update:

@0xMohamed and I, did an initial UI for the page, here is the design so far:

![image](https://github.com/egytech-fyi/egytech-fyi/assets/16765061/8651150a-7ec4-433b-ac42-c403be259bd2)


also, I created this form to get the best must know GitHub repos in Egypt:
https://tally.so/forms/mOP6kY/edit

Please feel free to fill this form.
